desktop_version/src/Script.cpp

```diff
@@ -2424,6 +2424,19 @@ void scriptclass::run(void)
                     position--;
                 }
             }
+            else if (words[0] == "setfont")
+            {
+#if !defined(NO_CUSTOM_LEVELS)
```

Comment:
I know `if !defined` is littered throughout the codebase but I'd like to fix this style sooner rather than later.
```suggestion
#ifndef NO_CUSTOM_LEVELS
```